ghost/core/test/e2e-api/members/webhooks.test.js

```diff
@@ -1680,17 +1684,22 @@ describe('Members API', function () {
             });
         });
 
-        it('Silently ignores an invalid offer id in metadata', async function () {
+        it('Supports creating an offer from a Stripe coupon', async function () {
```

Comment:
We had an explicit test for the opposite of this behaviour! At least it was half expected I guess 😁

Reply:
Haha yeah! Outside of migration flows, I think it's fair to assume that the offer will pre-exist in 99% of the cases and we don't need to "generate" it? Maybe a limitation we were aware of, but didn't want to complicate things for an edge case (outside of migrations) - don't know :)